packages/components/vite.config.ts

```diff
@@ -0,0 +1,33 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { resolve } from 'path';
+import { defineConfig } from 'vite';
+import dts from 'vite-plugin-dts';
+
+export default defineConfig({
+    plugins: [dts({ insertTypesEntry: true, rollupTypes: true })],
```

Comment:
```suggestion
    plugins: [react(), dts({ insertTypesEntry: true, rollupTypes: true })],
```
missing the vite plugin react?